src/stringIndices.js

```diff
@@ -0,0 +1,7 @@
+export const stringIndices = (string, index) => {
+  const arrString = string.split(' ')
```

Comment:
An error might be thrown for an empty string. Also, extra spaces would cause an error in obtaining the correct index. The following string 'Hello  there', has two spaces and the split method would cause an array to be formed in the manner['Hello', ' ', 'there'].